10/umbraco-workflow/getting-started/configuration.md

```diff
@@ -140,6 +143,22 @@ Sets the workflow type to one of Explicit (0), Implicit (1), or Exclude (2):
 - Implicit auto-approves steps where the author is a member of the approving group
 - Exclude behaves similar to Explicit, but excludes the original author from any notifications (that is the author can not approve their own work)
 
+#### ApprovalThreshold (int, requires a license)
+
+Sets the default approval threshold to one of One (0), Most (1), or All (2):
+
+- One: pending task requires approval from any member of the assigned approval group. This is the default behaviour for all installations (trial and licensed)
+- Most: pending task requires an absolute majority of group members. For example, a group with three members requires two approvals, a group with four members requires three approvals, and a group with six members requires four approvals.
+- All: pending task requires approval from all group members.
```

Comment:
Again, how about we put these into a table instead?
I think that would make it a little easier to read.
We could also add a column for the int value.